src/lib/utils/date.ts

```diff
@@ -1,2 +1,5 @@
 export const dateToString = (published: Date | string) =>
   new Date(published).toISOString().split("T")[0]
+export const formatDate = (date: Date, locale: string): string => {
+  return Intl.DateTimeFormat(locale).format(date)
+}
```

Comment:
We have `getLocaleTimestamp` inside `src/lib/utils/time.ts` which does this same thing with a bit more flexibility and type safety (allowing display options to be passed for example).

I think we should remove this addition and use `getLocaleTimestamp` instead

Reply:
```suggestion
```